src/bin/sleep.rs

```diff
@@ -3,19 +3,86 @@
 extern crate coreutils;
 
 use std::env;
+use std::io::{self, Write};
 use std::thread;
-use std::io;
+use std::time::Duration;
 
-use coreutils::extra::{OptionalExt, fail};
 
-#[allow(deprecated)]
+use coreutils::extra::{OptionalExt};
+
+static MAN_PAGE: &'static str = r#"NAME
+    sleep - delay for a specified amount of time
```

Comment:
`NAME` is actually there on the same line as `r#"` to prevent printing an extra newline.